README.md

```diff
@@ -1,8 +1,251 @@
+
+  `GITEA_URL`, `GITEA_USERNAME`, `GITEA_PASSWORD` are the values describing connection to the gitea server and credentials for the admin user.
+
+  If you have used [fuseml-installer](https://github.com/fuseml/fuseml) to set up the environment, there is already default gitea instance installed in your Kubernetes cluster. In such case set the values this way:
+  ```bash
+  export GITEA_URL=$(kubectl get VirtualService -n gitea gitea -o jsonpath="{.spec.hosts[0]}")
+  export GITEA_USERNAME=$(kubectl get Secret -n fuseml-workloads gitea-creds -o jsonpath="{.data.username}" | base64 -d)
+  export GITEA_PASSWORD=$(kubectl get Secret -n fuseml-workloads gitea-creds -o jsonpath="{.data.password}" | base64 -d)
+  ```
+  It is possible to use external gitea server instead. Make sure to provide correct environment variables.
+  
+  `TEKTON_DASHBOARD_URL` is the path to the Tekton server. As with other components, Tekton is installed by `fuseml-installer` into your cluster. To get the right URL, call
+  ```bash
+  export TEKTON_DASHBOARD_URL=$(kubectl get VirtualService -n tekton-pipelines tekton -o jsonpath="{.spec.hosts[0]}")
+  ```
+
+  Now it's possible to execute `bin/fuseml_core`.
+  Use the `--help` flag to get the command line options that you can supply. By default the server listens on the follwing ports: 8000 (http) and 8080 (grpc)
+
+* Run the client
+
+  Executing the client with `--help` option will show the usage instuctions
+```bash
+    > bin/fuseml_core-cli --help
+bin/fuseml_core-cli is a command line client for the FuseML API.
+
+Usage:
+    fuseml_core-cli [-host HOST][-url URL][-timeout SECONDS][-verbose|-v] SERVICE ENDPOINT [flags]
+
+    -host HOST:  server host (dev). valid values: dev, prod
+    -url URL:    specify service URL overriding host URL (http://localhost:8080)
+    -timeout:    maximum number of seconds to wait for response (30)
+    -verbose|-v: print request and response details (false)
+
+Commands:
+    application (list|register|get|delete)
+    codeset (list|register|get|delete)
+    runnable (list|register|get)
+    workflow (list|register|get|assign|list-runs)
+
+Additional help:
+    bin/fuseml_core-cli SERVICE [ENDPOINT] --help
+
+Example:
+    bin/fuseml_core-cli application list --type "predictor" --workflow "mlflow-sklearn-e2e"
+    bin/fuseml_core-cli codeset list --project "mlflow-project-01" --label "mlflow"
+    bin/fuseml_core-cli runnable list --id "ml-trainer-123" --kind "trainer" --labels '{
+          "function": "predict|train",
+          "library": "pytorch"
+       }'
+    bin/fuseml_core-cli workflow list --name "workflow"
+```
+
+  Without `-url` argument, client tries to connect to the server running locally at your machine. To find the URL of the fuseml-core server service that is installed in your Kubernetes cluster, call 
```

Comment:
We need to fix this, having to pass url is not really user friendly.

Reply:
https://github.com/fuseml/fuseml-core/pull/54